index.html

```diff
@@ -346,27 +347,48 @@ <h2>About myself</h2>
       </ul>
       <button class="aboutbutton">Get my Resume</button>
         <ul class="skillset">
-          <li>
-            <h3>languages</h3> <img class="arrow" src="./images/down.png" alt="rightarrow" />
+          <li class="languages">
+            <h3>languages</h3> <img class="arrow downnlanguage" src="./images/down.png" alt="rightarrow" />
+            <ul class="languageskills">
+              <li class="lang">
+                <img src="./images/language3.png" alt="js" />
+                <p>Javascript</p>
+              </li>
+              <li class="lang">
+                <img src="./images/language1.png" alt="js" />
+                <p>HTML</p>
+              </li>
+              <li class="lang">
+                <img src="./images/language2.png" alt="js" />
+                <p>CSS</p>
+              </li>
+            </ul>
           </li>
-          <ul class="languageskills">
-            <li class="lang">
-              <img src="./images/language3.png" alt="js" />
-              <p>Javascript</p>
-            </li>
-            <li class="lang">
-              <img src="./images/language1.png" alt="js" />
-              <p>HTML</p>
-            </li>
-            <li class="lang">
-              <img src="./images/language2.png" alt="js" />
-              <p>CSS</p>
-            </li>
-          </ul>
+          
           <li><h3>Frameworks </h3><img class="arrow" src="./images/right.png" alt="rightarrow" /></li>
           <li><h3>Skills </h3><img class="arrow" src="./images/right.png" alt="rightarrow" />
    </li>
         </ul>
     </section>
+    <footer class="contact-form">
+      <h2 class="h-contact">Contact me</h2>
+      <p class="contact-diescription">
+        If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+      </p>
+      <form action="#" method="post">
+        <ul>
+          <li>
+            <input type="text" required name="userName" maxlength="30" id="userName" placeholder="Joffrey"/>
+          </li>
+          <li>
+            <input type="email" required name="userEmail" id="userEmail" placeholder="joyjoffrey@gmail.com"/>
+          </li>
+          <li>
+            <textarea required maxlength="500" name="comment" id="comment" placeholder="Write your message here"></textarea>
+          </li>
+        </ul>
+        <button type="submit" class="contact-button">Get It Touch</button>
```

Comment:
- [ ] Kindly stick to the Figma design of the submit button. It seems a bit rounded and slim on your implementation as compared to the design.
#### Figma Design
![contact-submit-btn-shape-figma](https://user-images.githubusercontent.com/44057863/166581680-430948a0-3c9b-43ba-8fbe-6f056e9b4f7f.png)
#### Imp
![contact-submit-btn-shape-impl](https://user-images.githubusercontent.com/44057863/166581740-e32dbf65-e4a2-40ee-bb87-71de800c1c33.png)
l

Reply:
Good job in implementing the changes 👍